.env_example

```diff
@@ -0,0 +1,12 @@
+# Required
+NETWORK_ID=
+ETHEREUM_RPC_URL=
+
+# Optional
+MESH_ENDPOINT=
```

Comment:
Seems like we have a couple different naming conventions going on here, for example `ETHEREUM_RPC_URL` vs `MESH_ENDPOINT`